ink.scribl.Scribl.json

```diff
@@ -0,0 +1,39 @@
+{
+    "app-id": "ink.scribl.Scribl",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "19.08",
+    "sdk": "org.freedesktop.Sdk",
+    "sdk-extensions": ["org.freedesktop.Sdk.Extension.rust-stable"],
+    "command": "scribl",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--socket=pulseaudio"
+    ],
+    "build-options": {
+        "append-path": "/usr/lib/sdk/rust-stable/bin",
+        "env": {
+            "CARGO_HOME": "/run/build/scribl/cargo"
+        }
+    },
+    "modules": [{
+        "name": "scribl",
+        "sources": [{
+                "type": "git",
+                "url": "https://github.com/jneem/scribl",
+                "branch": "trunk"
+            },
+            "generated-sources.json"
+        ],
+        "buildsystem": "simple",
+        "build-commands": [
+            "cargo --offline fetch --manifest-path Cargo.toml --verbose",
+            "cargo --offline build --release --verbose",
+            "install -D target/release/scribl /app/bin/scribl",
+            "install -D metadata/ink.scribl.Scribl.appdata.xml /app/share/metainfo/ink.scribl.Scribl.appdata.xml",
+            "install -D metadata/ink.scribl.Scribl.svg /app/share/icons/scalable/apps/ink.scribl.Scribl.svg",
```

Comment:
```suggestion
            "install -D metadata/ink.scribl.Scribl.svg /app/share/icons/hicolor/scalable/apps/ink.scribl.Scribl.svg",
```

Reply:
Thanks! I also added some pre-rendered pngs, since the build seemed to complain about that.